helm/app/templates/dashboard-role.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:
+  name: {{ include "app.fullname" $ }}
+  namespace: {{ $.Release.Namespace }}
+  labels:
+    helm.sh/chart: {{ include "app.chart" $ }}
+    app.kubernetes.io/name: {{ include "app.name" $ }}
+    app.kubernetes.io/instance: {{ $.Release.Name }}
+    app.kubernetes.io/managed-by: {{ $.Release.Service }}
+rules:
+- apiGroups:
+  - "*"
+  resources:
+  - "*"
+  verbs:
+  - get
+  - list
+  - watch
+  - describe
+  - exec
+  - update
+  - delete
+- apiGroups:
+  - ""
+  resourceNames:
+  - dashboard-secret
```

Comment:
we should be able to pass secret name from values `.Values.kubernetes-dashboard. resourceNames ` ? or smth similar